pyproject.toml

```diff
@@ -433,6 +433,9 @@ filterwarnings = [
   "error::pytest.PytestUnraisableExceptionWarning",
   "ignore:.*ssl.SSLSocket:pytest.PytestUnraisableExceptionWarning",
   "ignore:.*ssl.SSLSocket:ResourceWarning",
+  # GH 44844: Can remove once minimum matplotlib version >= 3.7
```

Comment:
does this refer to both lines introduced here?

Reply:
Yup